src/cmd_line/commands/breakpoints.ts

```diff
@@ -0,0 +1,271 @@
+      }
+      const reqLine = this.delBreakpoint.line - 1;
+      const breakpoint = vscode.debug.breakpoints
+        .filter(isSourceBreakpoint)
+        .find(
+          (b) =>
+            b.location.uri.toString() === reqUri.toString() &&
+            b.location.range.start.line === reqLine
+        );
+      if (breakpoint) return vscode.debug.removeBreakpoints([breakpoint]);
+    } else if (isDelBreakpointFunction(this.delBreakpoint)) {
+      const functionName = this.delBreakpoint.function;
+      const breakpoint = vscode.debug.breakpoints
+        .filter(isFunctionBreakpoint)
+        .filter((b) => b.functionName === functionName);
+      if (breakpoint) return vscode.debug.removeBreakpoints(breakpoint);
+    } else if (isDelBreakpointHere(this.delBreakpoint)) {
+      const location = new vscode.Location(vimState.document.uri, vimState.cursorStartPosition);
+      const distFromLocationCharacter = (b: vscode.SourceBreakpoint) => Math.abs(b.location.range.start.character - location.range.start.character);
+
+      const breakpoint = vscode.debug.breakpoints
+        .filter(isSourceBreakpoint)
+        .filter(
+          (b) =>
+            b.location.uri.toString() === location.uri.toString() &&
+            b.location.range.start.line === location.range.start.line
+        ).sort((a, b) => distFromLocationCharacter(a) - distFromLocationCharacter(b))[0];
+      if (breakpoint) return vscode.debug.removeBreakpoints([breakpoint]);
+    }
+  }
+}
+
+/**
+ * List Breakpoints Command
+ */
+class ListBreakpointsCommand extends ExCommand {
+  public override isRepeatableWithDot: boolean = false;
+
+  async execute(vimState: VimState): Promise<void> {
+    type AnyBreakpoint = vscode.Breakpoint & vscode.SourceBreakpoint & vscode.FunctionBreakpoint;
+    const breakpoints = vscode.debug.breakpoints;
+    const lines = breakpoints.map((b, i) => {
+      const {
+        enabled,
+        condition,
+        location,
+        functionName,
+      } = b as AnyBreakpoint;
+      let entry = '';
+      entry += `#${i + 1}\t`;
+      entry += enabled ? '$(circle-filled)\t' : '$(circle-outline)\t';
+      entry += condition ? '$(debug-breakpoint-conditional)\t' : '\t';
+      entry += location ? `${path.basename(location.uri.fsPath)}:${location.range.start.line + 1}` : '';
+      entry += functionName ? `$(debug-breakpoint-function)${functionName}` : '';
+      return entry;
+    });
+    return vscode.window.showQuickPick(lines).then((selected) => {
+      if (selected) {
+        const [idWithHashtag] = selected.split('\t');
+        const id = parseInt(idWithHashtag.replace('#', ''), 10) - 1;
+        const breakpoint = breakpoints[id] as AnyBreakpoint;
+        if (breakpoint && breakpoint.location !== undefined) {
+          return vscode.window.showTextDocument(breakpoint.location.uri).then(
+            () => {
+              vimState.cursorStartPosition = breakpoint.location.range.start;
```

Comment:
@J-Fields couldn't get it to work while in the same file... Jumping to another file seems to be working.

Reply:
Hmm... try setting `vimState.cursorStopPosition`?

Reply:
Works!